guides/common/modules/proc_configuring-tracer-on-a-host.adoc

```diff
@@ -1,18 +1,31 @@
-[id="enabling-tracer-on-a-host_{context}"]
-= Enabling Tracer on a host
+[id="Configuring_Tracer_on_a_host_{context}"]
+= Configuring Tracer on a host
 
-Use this procedure to enable Tracer on {Project} and access Traces.
-Tracer displays a list of services and applications that need to be restarted.
+You can install Tracer on {ProjectName} and access Traces.
+Tracer displays a list of services and applications that are outdated and need to be restarted.
 Traces is the output generated by Tracer in the {ProjectWebUI}.
 
 .Prerequisites
+* The host is registered to {ProjectName}.
 include::snip_prerequisite-project-client-repository-enabled.adoc[]
 * Remote execution is enabled.
 
 .Procedure
+
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
 . On the *Traces* tab, click *Enable Traces*.
 . Select the provider to install `katello-host-tools-tracer` from the list.
++
+Alternatively, install Tracer using the CLI:
```

Comment:
do we really want to document this?

If so, it'd probably just be the same commands as the template. Also, "yum" only works for EL7+.

Reply:
We can, but then we should use the proper heading for a CLI procedure:

```suggestion
[id="cli-Configuring..."]
.CLI procedure
```

Reply:
But with ".CLI procedure", we generally refer to Foreman/Smart Proxies, not commands on hosts. I like that there's abstraction for the commands on hosts through the REX job templates.

Reply:
The CLI procedure/alternative step is too confusing and is unnecessary, so I removed it.